src/era_5g_relay_network_application/era_5g_relay_network_application/worker.py

```diff
@@ -36,11 +39,13 @@ def stop(self):
     def get_data(self):
         try:
             d = self.queue.get(block=True, timeout=1)
+            if type(self.inst) == LaserScan:
```

Comment:
Not a real issue here, but in general, it's better to use `isinstance(self.inst, LaserScan)`.